livre/core/render/GLContext.h

```diff
@@ -33,7 +33,7 @@ namespace livre
 class GLContext
 {
 public:
-    LIVRECORE_API GLContext();
+    LIVRECORE_API GLContext( const GLEWContext* glewContext );
```

Comment:
GLContext object should not be forced to have a glew context. When we added the static function with @tribal-tec we though it was a convenient place to add. And, since we are having always same capability GPUs you rarely need multiple contexts. Can you at least keep a default contructor ?